src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -2218,7 +2219,7 @@ private void AddDllFile(string filename)
             {
                 // we're not sure if this is a managed assembly or not
                 // we try to load it, if it fails - we add it as an additional file
-                var result = PackageLoader.TryLoadFrom(filename, out Assembly assem);
+                var result = PackageLoader.TryLoadFrom(AssemblyLoadContext.Default, filename, out Assembly assem);
```

Comment:
Not sure if this needs to be isolated or not.

Reply:
oh - this is just for publishing, I mean, I still think what I stated is the way to go, but maybe less important.

Reply:
I might add a new task for this